### ISSUE TITLE
gsoc 2019

### ISSUE THREAD

**Issue by ashwath7500** [salesforce/gsoc#7]
Hey im interested in contributing to transmorgrifAi during gsoc 2019 , how do i get started?? I am not familiar with scala and apache spark

**Comment by tanhengyeow** [comment]
@ashwath7500  Hi, I would suggest reading the [docs](https://docs.transmogrif.ai/en/stable/) to get familiar with the project. After that, I would say this [list](https://github.com/salesforce/TransmogrifAI/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) would be a good starting point to get familiar with the project's commit workflow and also allow one to dive a little into the codebase.

**Comment by tanhengyeow** [comment]
In my opinion, not being familiar with the technologies involved isn't an issue as long as you are willing to learn them. Nevertheless, knowing them beforehand definitely helps you to understand the codebase faster 😛

**Comment by ashwath7500** [comment]
Thanks for your response. I will definitely do my level best for your
organization.

**Comment by tanhengyeow** [comment]
@ashwath7500 Oh I'm not an employee for Salesforce at this point, I'm a GSoC aspirant just like you 😛

**Comment by ashwath7500** [comment]
you are also trying for transmorgifAi ?

**Comment by tanhengyeow** [comment]
I'm aiming to work for Design System React 😄

**Comment by rmeena840** [comment]
@tanhengyeow Cool. I am also GSoC aspirant. Have you found any discussion forum Design System React?

**Comment by tanhengyeow** [comment]
I don't think so, at least not that I know of. I guess all communication would be encouraged here!

**Comment by rmeena840** [comment]
Okay. I tried reaching out the mentors but no response yet.

**Comment by aswinshenoy** [comment]
I too tried contacting them over email, seems like they are pretty engaged with their regular salesforce works. I think they will be much active when it is nearing to the application time.
